charms/docker/compose.py

```diff
@@ -47,6 +47,19 @@ def kill(self, service=None):
             cmd = "docker-compose kill"
         self.run(cmd)
 
+    def rm(self, service=None):
+        '''
+        Convenience method that wraps `docker-compose rm`
+
+        usage: c.rm('nginx') to remove the 'nginx' service from the
```

Comment:
nevermind, I just realized that docker-compose rm -f without a service is an expected construct.